docs2/site/docs/migrations/migration4.md

```diff
@@ -214,6 +214,7 @@ Here are some of the situations that you may run into with version 4:
 - `EnumGraphType` is stricter, requiring internal values for serialization, and external values for deserialization
 - `IdGraphType` (which allows any basic type) does not coerce variable values to trimmed strings during deserialization
 - `IdGraphType` does not trim serialized values (but does convert them to strings)
+- `DateTimeGraphType` serializes values to strings instead of letting the JSON serializer do so
```

Comment:
And it looks correct. Serializer settings should not necessarily affect (although they **can**) all returned results. Some scalars have _fixed_ format by their nature. Perhaps it is worth adding a note about it to the documentation.

Reply:
:+1: if you want to suggest a change to the docs, that'd be great.  I think my brain is fried right now, working on so much stuff at once!

We can probably just leave it though.  We are nearing the end of the 4.x lifecycle.

Reply:
#2842